paparazzi/src/test/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtensionTest.kt

```diff
@@ -77,7 +85,7 @@ class AccessibilityRenderExtensionTest {
             relativePath = expected.path,
             image = image,
             goldenImage = ImageIO.read(expected),
-            maxPercentDifferent = 0.2,
+            maxPercentDifferent = 0.1, // Default percent defined in paparazzi.
```

Comment:
can this be set to 0 now?

Reply:
unfortunately there is a small diff between what mac generates and what is generated on windows. ([build](https://github.com/cashapp/paparazzi/actions/runs/1477112544))

![delta-accessibility](https://user-images.githubusercontent.com/18534850/142450497-2217b72f-3610-45e9-9aca-b695792e552c.png)

For now I will leave this at 0.1